### PR TITLE
[INLONG-9913][Sort] Solve end-to-end-tests-v1.13 naming problem and test case run failed.

### LINKED ISSUE #9913: [Improve][Sort] Solve end-to-end-tests-v1.13 naming problem and test case run failed.
### Description

There is not workflow file for flink1.13 test case.I wish to add it.

image

The completed test case was not tested when push to repository, because of their name.And those case have a little problem when  test them.So, I want to solve them.

J8A8% 0_H1M@%S VY)ATDBV

{~NF 4(T 83I2DH%@B$A833

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by XiaoYou201**
### Prepare a Pull Request

- [INLONG-9913][Sort] Solve end-to-end-tests-v1.13 naming problem and test case run failed.

- Fixes #9913 

### Motivation

The workflow files directory not have ut file for flink v1.13, So I want add it. And There are some problem which end-to-end-v1.13 test case running. I repaired it.Hope the project will get better and better.

### Modifications

add the ci_ut_flink13.yml, rename test case and modify some sql and config file.

### CODE REVIEW

File: inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/utils/MySqlContainer.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public class MySqlContainer extends JdbcDatabaseContainer {
     public static final String IMAGE = "mysql";
     public static final Integer MYSQL_PORT = 3306;
 
-    private static final String MY_CNF_CONFIG_OVERRIDE_PARAM_NAME = "MY_CNF";
+    private static final String MY_CNF_CONFIG_OVERRIDE_PARAM_NAME = "TC_MY_CNF";
```
Comment (EMsnap) [inline inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/utils/MySqlContainer.java:38]: excuse me, any reasons  for change config name ?

File: inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/Mysql2KafkaTest.java [hunk 2]
```diff
@@ -165,7 +166,7 @@ public void testKafkaWithSqlFile() throws Exception {
                     "INSERT INTO test_input "
                             + "VALUES (1,'jacket','water resistent white wind breaker',0.2, null, null, null);");
             stat.execute(
-                    "INSERT INTO test_input VALUES (2,'scooter','Big 2-wheel scooter ',5.18, null, null, null);");
+                    "INSERT INTO test_input VALUES (2,'scooter','Big 2-wheel scooter',5.18, null, null, null);");
```
Comment (EMsnap) [inline inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/Mysql2KafkaTest.java:169]: the extra blank will influence the test result ?

File: inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/utils/MySqlContainer.java (hunk 1 above)
Comment (XiaoYou201) [reply inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/utils/MySqlContainer.java:38]: Mysql initialization will use this fixed name field  when the testcontainers run mysql docker contaner.

As https://java.testcontainers.org/modules/databases/mysql/ instruction

File: inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/Mysql2KafkaTest.java (hunk 2 above)
Comment (XiaoYou201) [reply inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/Mysql2KafkaTest.java:169]: No, it has no influence fot the test result.But this extra blank space may cause misunderstandings.So, I deleted it.

**Review by EMsnap — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**